scripts/setup/apt-repos/zulip/bookworm.list

```diff
@@ -1,2 +1,8 @@
 deb http://apt.postgresql.org/pub/repos/apt/ bookworm-pgdg main
 deb-src http://apt.postgresql.org/pub/repos/apt/ bookworm-pgdg main
+
+deb https://ppa.launchpadcontent.net/strukturag/libheif/ubuntu/ bookworm main
+deb-src https://ppa.launchpadcontent.net/strukturag/libheif/ubuntu/ bookworm main
+
+deb https://ppa.launchpadcontent.net/strukturag/libde265/ubuntu/ bookworm main
+deb-src https://ppa.launchpadcontent.net/strukturag/libde265/ubuntu/ bookworm main
```

Comment:
The PPA doesn't have `bookworm` builds, so these lines are wrong.

Reply:
We do not have a repo that we host.  I think we could [pull from `bookworm-backports`]( https://packages.debian.org/bookworm-backports/source/libheif)?